eng/tools/mcp/azure-sdk-python-mcp/main.py

```diff
@@ -249,7 +249,33 @@ def init_local_tool(tsp_config_path: str, repo_path:str) -> Dict[str, Any]:
             "stderr": "",
             "code": 1
         }
+    
+@mcp.tool("update")
+def update_tool(commit_hash: str, package_path:str) -> Dict[str, Any]:
+    """Initializes and subsequently generates a typespec client library directory from a local azure-rest-api-specs repo.
+
+    This command is used to update a client library to a specific commit hash in the azure-rest-api-specs repository.
+
+    Args:
+        commit_hash: The commit hash to update to.
+        package_path: The path to the directory of the tsp-location.yaml (i.e. ./azure-sdk-for-python/sdk/eventgrid/azure-eventgrid).
```

Comment:
This should be optional. What if someone is asking the agent to update their client library from the same client library directory? Maybe we should check if tsp-location.yaml is in the same directory first, then if it isnt we can ask for the package path